packages/bot-web-ui/src/components/dashboard/tour-slider.tsx

```diff
@@ -138,12 +121,22 @@ const TourSlider = ({
             }
         });
     }, [step]);
-    const content_data = BOT_BUILDER_MOBILE.find(({ key }) => key === step);
 
-    const onClickNext = React.useCallback(() => {
+    const onChange = (param: string) => {
+        if (
+            param === 'inc' &&
+            step < Object.keys(!has_started_onboarding_tour ? BOT_BUILDER_MOBILE : DBOT_ONBOARDING_MOBILE).length
```

Comment:
can we put it in a variable to the `if` condition won't get stretched like this

Reply:
done  ✅